README.md

```diff
@@ -5,21 +5,21 @@ __Fetches concepts from concept-transformers and adds them to kafka.__
 
 ## Installation
 
-`go get github.com/Financial-Times/concept-publisher`
-
-## Running locally
-
 ```
-go build
+go get -u github.com/kardianos/govendor
+go get -u github.com/Financial-Times/concept-publisher
```

Comment:
A very minor comment. This command is going to download the dependencies, which potentially can create mess in your `GOPATH` with vendoring. 
Probably it make sense to substitute this line with:
`git clone git@github.com:Financial-Times/concept-publisher.git`